php/nette/composer.json

```diff
@@ -0,0 +1,10 @@
+{
+    "require": {
+        "nette/application": "^3.0",
+        "nette/bootstrap": "^3.0",
+        "nette/di": "^3.0"
+    },
+    "autoload": {
```

Comment:
What is this for ?

Reply:
this is for loading all classes in `app` directory using composer, I am pretty sure that application will not work without it

Reply:
ah, it is not related to `composer autoload` (which is done in `Dockerfile`)

it was probably a noob question, I am more a `rubyist` that a `php` adopter